src/native/fs/fs_napi.cpp

```diff
@@ -560,6 +571,7 @@ struct FSWorker : public Napi::AsyncWorker
     double _took_time;
     Napi::FunctionReference _report_fs_stats;
     bool _should_add_thread_capabilities;
+    std::vector<gid_t> _supplemental_groups;
```

Comment:
looks like the loading supplemental_groups to the fs_context is missing in the CLI, did I miss anything? 
it should be added in native_fs_utils.get_fs_context()

Reply:
please make sure it's being passed to NAPI correctly in all places, especially in NamespaceFS, if you could add tests on these flows as well it'll be even better